libraries/ms-helper/inc/exported_enums.h

```diff
@@ -25,7 +25,37 @@ typedef enum {
   NUM_EE_CHARGER_CONN_STATES,
 } EEChargerConnState;
 
-typedef enum {
+typedef enum EELightType {
+  EE_LIGHT_TYPE_HIGH_BEAMS = 0,
+  EE_LIGHT_TYPE_LOW_BEAMS,
+  EE_LIGHT_TYPE_DRL,
+  EE_LIGHT_TYPE_BRAKES,
+  EE_LIGHT_TYPE_SIGNAL_RIGHT,
+  EE_LIGHT_TYPE_SIGNAL_LEFT,
+  EE_LIGHT_TYPE_SIGNAL_HAZARD,
+  EE_LIGHT_TYPE_STROBE,
+  NUM_EE_LIGHT_TYPES
```

Comment:
clang-format behaves better if you always add a trailing comma

Reply:
:b:ool